aws-provider/postgresdb/README.md

```diff
@@ -0,0 +1,346 @@
+# `crossplane` PostgresDB example
+
+In this more complex example, we're going to walk through the process of
+enabling provider-families with `sts:AssumeRoleWithWebIdentity` to the
+management cluster, then using that to bootstrap a database using a secondary
+role assumed by crossplane for the actual build.
+
+> **Warning** Creating IAM permissions
+>
+> The  following document assumes that you have permissions to create IAM roles
+> and policies inside your account.
+>
+> If you do not have such permissions, please reach out to your account
+> administrator and ask them to create these for you.
```

Comment:
```suggestion
__Warning__: The following document assumes that you have permission to create IAM roles and policies inside your account. If you do not have such permissions, please reach out to your account administrator and ask them to create these for you.
```
Following the same syntaxis as our docs?

Reply:
Checking the other clarification blocks you have below, I tend to think using quote block it is better when is a multi-paragraph block, so ignore me